frontend/src/components/Pages/MissionPage/MapPosition/MissionMapView.tsx

```diff
@@ -48,16 +49,32 @@ export const MissionMapView = ({ mission }: MissionProps) => {
 
     const imageObjectURL = useRef<string>('')
 
+    const FetchMapInfo = (mission: Mission) => {
+        const map = useQuery({
+            queryKey: [mission.missionId],
+            queryFn: async () => {
+                const missionDefinition = await BackendAPICaller.getMissionDefinitionById(mission.missionId!)
+                return missionDefinition.map
+            },
+            retryDelay: 60 * 1000,
+            staleTime: 60 * 1000,
```

Comment:
These things should probably be abstracted away, but no need to do it in this PR